### ISSUE TITLE
Does not display well with Breeze Dark on Plasma 5

### ISSUE THREAD

**Issue by madpipeline** [ritze/kmail-header-material-reloaded#1]
![imagine2](https://cloud.githubusercontent.com/assets/3527534/6450762/ca039356-c132-11e4-98f2-3ee877444763.png)

The background color should change according to the Desktop theme.

**Comment by ritze** [comment]
I don't think that I can request the kde5 theme. The capabilities of a kmail message theme are limited.

The simplest way should be to copy that theme and replace the colors with dark colors.

**Comment by madpipeline** [comment]
That would require another theme.

How do the default themes resolve this?

**Comment by ritze** [comment]
The default themes are no "messageviewer themes".

There are only two example themes: "Example" and "Example fancy". These themes don't define a background color.

I don't know how I get a default color in css or calculate a new color with the old default color.
